bitnami/mongodb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 8.1.3
+version: 8.1.4
```

Comment:
Could you bump the minor one version? As this is a new feature
```suggestion
version: 8.2.0
```

Reply:
Sure.